### ISSUE TITLE
 GPL Ghostscript not working

### ISSUE THREAD

**Issue by FrequentErvin** [vectorgraphics/asymptote#362]
I'm relatively new to Asymptote. I've used it through Overleaf (begin{asy} ...) and imported manually every package I needed. For example, if I needed to import cse5 I would get the raw file from GitHub and input it into my working directory. I've tried installing Asymptote locally but failed when I got to GPL Ghostscript since it's outdated / not available / not working when I try and run asy. I was following this installation guide: https://asymptote.sourceforge.io/ 

TL;DR: I want to run  Asymptote locally, but can't get GPL Ghostscript to work.

**Comment by johncbowman** [comment]
Install AGPL Ghostscript 10.0.0 for your platform from
https://www.ghostscript.com/releases/gsdnld.html
